libs/service-portal/settings/personal-information/src/lib/navigation.ts

```diff
@@ -0,0 +1,44 @@
+import { PortalNavigationItem } from '@island.is/portals/core'
+import { m } from '@island.is/service-portal/core'
+import { PersonalInformationPaths } from './paths'
+
+export const personalInformationNavigation: PortalNavigationItem = {
+  name: m.settings,
+  navHide: true,
+  children: [
+    // {
```

Comment:
Not part of your PR I know, but this should be removed and just added when needed :)

Reply:
Yesss, I agree with you will clean this up a bit.